src/coreclr/gc/unix/numasupport.h

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
While you are on it, you can also delete src\coreclr\pal\src\include\pal\numa.h and `NUMASupportCleanup` call in src\coreclr\pal\src\init\pal.cpp.